MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -214,6 +214,26 @@ public string VideoDoViTitle
             }
         }
 
+        /// <summary>
+        /// Gets the audio spatial format.
+        /// </summary>
+        /// <value>The audio spatial format.</value>
+        public AudioSpatialFormat AudioSpatialFormat
```

Comment:
This would need to be nullable for backwards compatibility?

@nielsvanvelzen

Reply:
Should be nullable or have a default value

Reply:
I had tried to make it nullable before but it was problematic. What do you mean by a default value? I made it a getter like `VideoRange` and `VideoRangeType`, both of which are also enums with a similar zero value like `None`.

Reply:
Adding this attribute to this property changes the api spec
`[DefaultValue(AudioSpatialFormat.None)]`

```diff
"AudioSpatialFormat": {
	"enum": [
	  "None",
	  "DolbyAtmos",
	  "DTSX"
	],
	"allOf": [
	  {
		"$ref": "#/components/schemas/AudioSpatialFormat"
	  }
	],
	"description": "Gets the audio spatial format.",
+	"default": "None",
	"readOnly": true
},
```